assets/src/edit-story/components/canvas/singleSelectionMoveable/useDrag.js

```diff
@@ -94,7 +94,7 @@ function useSingleSelectionDrag({
 
     // When dragging finishes, set the new properties based on the original + what moved meanwhile.
     const [deltaX, deltaY] = frame.translate;
-    if (deltaX !== 0 || deltaY !== 0 || isDropSource(selectedElement.type)) {
+    if (deltaX !== 0 || deltaY !== 0) {
```

Comment:
Hm, what is the reason for this test existing in the first place and can it really safely be removed?

Reply:
I believe it was incorrectly added, doesn't seem to be any reason why to add this check to this `if`, it can be done below as well. If the dragging didn't result in any movement (deltas both 0) then there is no dropping happening anyway.